robusta_krr/strategies/limit.py

```
@@ -0,0 +1,138 @@
+        # Here we filter out pods from calculation that have less than `points_required` data points
+        filtered_data = {
+            pod: values for pod, values in data.items() if data_count.get(pod, 0) >= self.settings.points_required
+        }
+
+        if len(filtered_data) == 0:
+            return ResourceRecommendation.undefined(info="Not enough data")
+
+        if object_data.hpa is not None and object_data.hpa.target_cpu_utilization_percentage is not None:
+            return ResourceRecommendation.undefined(info="HPA detected")
+
+        cpu_usage_request = self.settings.calculate_cpu_request(filtered_data)
```

Flake8 (7.3.0)

[error] local variable 'cpu_usage_request' is assigned to but never used

(F841)